modules/contentbox/modules/contentbox-admin/modules/contentbox-filebrowser/handlers/Home.cfc

```diff
@@ -44,6 +44,8 @@ component{
 		boolean widget=false,
 		struct settings={}
 	){
+		if(fileExists(expandPath('\modules\contentbox\content\download.zip')))
```

Comment:
What is the purpose of this?

Reply:
I had been using this for my debugging purpose. I've now removed that. You can check that in this PR af1490f.